### DISCUSSION TITLE
ubuntu vm on mac m1

### DISCUSSION THREAD

**Discussion by SquishyGlue3** [pieces-app/support#276] [Q&A]
hello
can not download this to my vm as it is arm based architecture 
can someone help me?

**Comment by mack-at-pieces** [comment]
Hey @SquishyGlue3 - Thanks so much for reaching out here. Sorry you're running into problems getting Pieces up and running on your Ubuntu VM.

Our lead linux eng is OOO this week and will be back on Monday and will be more than happy to take a look into your setup to see what might be causing the issue.

If you shoot me an email at mack@pieces.app we can set up a time to connect if that works for you!

Thanks!
